docs/en/ingest-management/elastic-agent/debug-standalone-elastic-agent.asciidoc

```diff
@@ -0,0 +1,203 @@
+[[debug-standalone-agents]]
+= Debug standalone {agent}s
+
+When you run standalone {agent}s, you are responsible for monitoring the status
+of your deployed {agent}s. You cannot view the status or logs in {fleet}.
+
+Use the following tips to help identify potential issues.
+
+Also refer to <<fleet-troubleshooting>> for guidance on specific problems.
+
+NOTE: You might need to log in as a root user (or Administrator on Windows) to
+run these commands.
+
+[discrete]
+== Check the status of the running {agent}
+
+To check the status of the running {agent} daemon and other processes managed by
+{agent}, run the `status` command. For example:
+
+[source,shell]
+----
+elastic-agent status
+----
+
+Returns something like: 
+
+[source,yaml]
+----
+Status: HEALTHY
+Message: (no message)
+Applications:
+  * metricbeat  (HEALTHY)
+                Running
+  * filebeat    (STOPPING)
+                (no message)
+----
+
+By default, this command returns the status in human-readable format. Use the
+`--output` flag to change it to `json` or `yaml`.
+
+For more information about this command, refer to
+<<elastic-agent-status-command>>.
+
+//REVIEWERS: Looks like JSON and YAML return status codes instead of healthy,
+//stopping, etc. Where can I find the mapping of status code to status?
```

Comment:
I think it's defined in the protob file: https://github.com/elastic/beats/blob/522c8624fe83d4a810af2d02bf3ecf6e10d0fa80/x-pack/elastic-agent/pkg/agent/control/proto/control.pb.go#L35-L44

Reply:
@michel-laterman Thanks. BTW, it seems weird that when I run the `status` command, the response _sometimes_ includes the status in the message field, but sometimes it does not. That's my rationale for documenting the status codes. Let me know if this behavior is expected or a bug.

Here's an example of a response:

```
{
    "Status": 2,
    "Message": "",
    "Applications": [
        {
            "ID": "filebeat--7.17.0-a1bd5481",
            "Name": "filebeat",
            "Status": 0,
            "Message": "Starting",
            "Payload": null
        },
        {
            "ID": "metricbeat--7.17.0-732bc2d2",
            "Name": "metricbeat",
            "Status": 5,
            "Message": "",
            "Payload": null
        }
    ]
}
```

Reply:
@michel-laterman is `the response sometimes includes the status in the message field, but sometimes it does not.` a known bug? Maybe the process aren't started yet?

Reply:
@michel-laterman Can you open an issue if this is a bug? I'm going to go ahead and merge/backport this PR.